src/tests/java/picard/illumina/parser/CycleIlluminaFileMapTest.java

```diff
@@ -75,42 +75,4 @@ private static String constructPathString(int lane, int cycle) {
             },
         };
     }
-
-  /*  @Test(dataProvider = "iteratorTestData")
```

Comment:
I think this was commented out during a refactor that removed CycleFilesIterator. CycleIlluminFileMap is tested in other unit tests so I agree with @yfarjoun that this file can be removed.